From 9adb32493608e3865c9145cdbb991b552d21863e Mon Sep 17 00:00:00 2001
From: Ayush Dey <deyayush6@gmail.com>
Date: Tue, 16 Sep 2025 12:04:39 +0530
Subject: [PATCH 15/17] input: add vlctest_ protocol

Add vlctest_ protocol as a file type. This will be required
in the next commit for using a dummy access module with the
libvlc_downloader.
---
 src/input/item.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/input/item.c b/src/input/item.c
index f9949cf83c..90d430a566 100644
--- a/src/input/item.c
+++ b/src/input/item.c
@@ -1242,6 +1242,7 @@ static enum input_item_type_e GuessType( const input_item_t *p_item, bool *p_net
         { "v4l",    ITEM_TYPE_CARD, false },
         { "vcd",    ITEM_TYPE_DISC, false },
         { "vdr",    ITEM_TYPE_STREAM, true },
+        { "vlctest_",ITEM_TYPE_FILE, false },
         { "wasapi", ITEM_TYPE_CARD, false },
         { "window", ITEM_TYPE_CARD, false },
     };
-- 
2.50.1 (Apple Git-155)

